From: Horst Beepmanh <>
Subject: Include-file shims

Generally in line with my tolerance of theoretical impurity. Some
"header" files had their name changed, and this allows them to be
accessed using the old name.
---
--- /dev/null
+++ b/Interfaces/AIncludes/ATalkEqu.a
@@ -0,0 +1 @@
+	INCLUDE		'AppleTalk.a'

--- /dev/null
+++ b/Interfaces/AIncludes/ATalkEqu.a.idump
@@ -0,0 +1 @@
+TEXTMPS 
\ No newline at end of file

--- /dev/null
+++ b/Interfaces/AIncludes/PackMacs.a
@@ -0,0 +1 @@
+	INCLUDE		'Packages.a'

--- /dev/null
+++ b/Interfaces/AIncludes/PackMacs.a.idump
@@ -0,0 +1 @@
+TEXTMPS 
\ No newline at end of file

--- /dev/null
+++ b/Interfaces/AIncludes/PaletteEqu.a
@@ -0,0 +1 @@
+	INCLUDE		'Palettes.a'

--- /dev/null
+++ b/Interfaces/AIncludes/PaletteEqu.a.idump
@@ -0,0 +1 @@
+TEXTMPS 
\ No newline at end of file

--- /dev/null
+++ b/Interfaces/AIncludes/PrEqu.a
@@ -0,0 +1 @@
+	INCLUDE		'Printing.a'

--- /dev/null
+++ b/Interfaces/AIncludes/PrEqu.a.idump
@@ -0,0 +1 @@
+TEXTMPS 
\ No newline at end of file

--- /dev/null
+++ b/Interfaces/AIncludes/PrintCallsEqu.a
@@ -0,0 +1 @@
+	INCLUDE		'PrintTrapsEqu.a'

--- /dev/null
+++ b/Interfaces/AIncludes/PrintCallsEqu.a.idump
@@ -0,0 +1 @@
+TEXTMPS 
\ No newline at end of file

--- /dev/null
+++ b/Interfaces/AIncludes/QDOffscreenEqu.a
@@ -0,0 +1 @@
+	INCLUDE		'QDOffscreen.a'

--- /dev/null
+++ b/Interfaces/AIncludes/QDOffscreenEqu.a.idump
@@ -0,0 +1 @@
+TEXTMPS 
\ No newline at end of file

--- /dev/null
+++ b/Interfaces/AIncludes/QuickEqu.a
@@ -0,0 +1 @@
+	INCLUDE		'QuickDraw.a'

--- /dev/null
+++ b/Interfaces/AIncludes/QuickEqu.a.idump
@@ -0,0 +1 @@
+TEXTMPS 
\ No newline at end of file

--- /dev/null
+++ b/Interfaces/AIncludes/SCSIEqu.a
@@ -0,0 +1 @@
+	INCLUDE		'SCSI.a'

--- /dev/null
+++ b/Interfaces/AIncludes/SCSIEqu.a.idump
@@ -0,0 +1 @@
+TEXTMPS 
\ No newline at end of file

--- /dev/null
+++ b/Interfaces/AIncludes/ScriptEqu.a
@@ -0,0 +1 @@
+	INCLUDE		'Script.a'

--- /dev/null
+++ b/Interfaces/AIncludes/ScriptEqu.a.idump
@@ -0,0 +1 @@
+TEXTMPS 
\ No newline at end of file

--- /dev/null
+++ b/Interfaces/AIncludes/ShutDownEqu.a
@@ -0,0 +1 @@
+	INCLUDE		'ShutDown.a'

--- /dev/null
+++ b/Interfaces/AIncludes/ShutDownEqu.a.idump
@@ -0,0 +1 @@
+TEXTMPS 
\ No newline at end of file

--- /dev/null
+++ b/Interfaces/AIncludes/SlotEqu.a
@@ -0,0 +1 @@
+	INCLUDE		'Slots.a'

--- /dev/null
+++ b/Interfaces/AIncludes/SlotEqu.a.idump
@@ -0,0 +1 @@
+TEXTMPS 
\ No newline at end of file

--- /dev/null
+++ b/Interfaces/AIncludes/ToolEqu.a
@@ -0,0 +1 @@
+	INCLUDE		'ToolUtils.a'

--- /dev/null
+++ b/Interfaces/AIncludes/ToolEqu.a.idump
@@ -0,0 +1 @@
+TEXTMPS 
\ No newline at end of file

--- /dev/null
+++ b/Interfaces/AIncludes/VideoEqu.a
@@ -0,0 +1 @@
+	INCLUDE		'Video.a'

--- /dev/null
+++ b/Interfaces/AIncludes/VideoEqu.a.idump
@@ -0,0 +1 @@
+TEXTMPS 
\ No newline at end of file

--- /dev/null
+++ b/Internal/Asm/DecompressorPatchEqu.a
@@ -0,0 +1 @@
+	INCLUDE		'Decompression.a'

--- /dev/null
+++ b/Internal/Asm/DecompressorPatchEqu.a.idump
@@ -0,0 +1 @@
+TEXTMPS 
\ No newline at end of file
-- 
2.18.0

